### PR TITLE
closes #7

### LINKED ISSUE #7: Backend Model: Product_Review
Product_Review
- id
- productId (parent)
- userId (parent)
- body (text) 
- stars( integers )

### PR DESCRIPTION
**Pull request by edwardtayloehill**
closes #7

### CODE REVIEW

File: db/models/index.js [hunk 1]
```diff
@@ -7,6 +7,9 @@
 const User = require('APP/db/models/user');
 const Product = require('APP/db/models/product');
 const Address = require('APP/db/models/address');
+const Product_Review = require('APP/db/models/product_review')
 
 Address.belongsTo(User)
+Product_Review.belongsTo(User)
+Product_Review.belongsTo(Product)
 module.exports = { User, Product, Address };
```
Comment (harryttd) [inline db/models/index.js]: I think you guys forgot to export reviews.

File: tests/product_review.test.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+const db = require('APP/db');
+const Product_Review = require('APP/db/models/product_review');
+const expect = require('chai').expect;
+
+describe('Product Model', () => {
```
Comment (harryttd) [inline tests/product_review.test.js]: Good testing guys. It seems you are only testing for validation errors but not that the database actually contains data that you created an instance with. Shouldn't you also test that the build was successful and actually contains the data you fed it?

File: db/models/product_review.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    validate: {
+      notEmpty: true
+    }
+  },
+  stars: {
+    type: Sequelize.INTEGER,
+    allowNull: false,
+    defaultValue: 0,
+    validate: {
+      is: '/\d(?![0-5]\S)/g',
```
Comment (harryttd) [inline db/models/product_review.js]: if you want a single digit between 1 and five you can just do: /^[0-5]/
This is saying the beginning of a line should contain one digit between 0 and 5.
I don't know if regex will validate properly here though as the data type is an integer.

File: db/models/index.js (hunk 1 above)
Comment (edwardtayloehill) [inline db/models/index.js]: Got it!

File: db/models/product_review.js (hunk 3 above)
Comment (edwardtayloehill) [inline db/models/product_review.js]: fug regex - switched to an enum

**Pull request merged**